Without this patch, `make clean` removes man pages and then the 64-bit
build tries to re-create them with tools that aren't in OmniOS.

diff -wpruN --no-dereference '--exclude=*.orig' a~/Makefile a/Makefile
--- a~/Makefile	1970-01-01 00:00:00
+++ a/Makefile	1970-01-01 00:00:00
@@ -64,13 +64,13 @@ UTILS = $(INSTALLABLE) \
 LDLIBS=libgif.a -lm
 
 MANUAL_PAGES = \
-	doc/gif2rgb.xml \
-	doc/gifbuild.xml \
-	doc/gifclrmp.xml \
-	doc/giffix.xml \
-	doc/giflib.xml \
-	doc/giftext.xml \
-	doc/giftool.xml
+	doc/gif2rgb.1 \
+	doc/gifbuild.1 \
+	doc/gifclrmp.1 \
+	doc/giffix.1 \
+	doc/giflib.7 \
+	doc/giftext.1 \
+	doc/giftool.1
 
 SOEXTENSION	= so
 LIBGIFSO	= libgif.$(SOEXTENSION)
@@ -119,7 +119,7 @@ clean:
 	rm -f $(UTILS) $(TARGET) libgetarg.a libgif.a $(LIBGIFSO) libutil.a $(LIBUTILSO) *.o
 	rm -f $(LIBGIFSOVER)
 	rm -f $(LIBGIFSOMAJOR)
-	rm -fr doc/*.1 *.html doc/staging
+	#rm -fr doc/*.1 *.html doc/staging
 
 check: all
 	$(MAKE) -C tests
